### ISSUE TITLE
Add support for providing a list of skills to review at the end of a question assessment

### ISSUE THREAD

**Issue by BenHenning** [oppia/oppia-android#247]
Learners can receive a list of skills to review after completing a question assessment. This is supported by the domain API for the QuestionAssessmentProgressController, but the implementation does not yet generate this list.

**Comment by viktoriias** [comment]
Can I work on this?

**Comment by rt4914** [comment]
@BenHenning PTAL.

**Comment by BenHenning** [comment]
Deferring to @vinitamurthi to determine; she's managing the questions work.

**Comment by vinitamurthi** [comment]
I believe @TheRealJessicaLi is already working on this. 
I think this should be art end up being part of the results page that @TheRealJessicaLi  and @mschanteltc  are working on. Jessica can you take a look at this feature and work with chantel?

**Comment by TheRealJessicaLi** [comment]
Yep, this will be part of the question scoring feature, and it's blocked by the score/mastery calculation logic. @vinitamurthi I'll be setting up a meeting with Chantel to discuss the UI, so I'll make sure to bring this up then!

**Comment by BenHenning** [comment]
Given that questions are on hold, closing this for now. Much of the calculation work has been checked in and I suspect it will be relevant when we revisit assessments & mastery in the future.

**Comment by BenHenning** [comment]
Per the TODO, let's finish the rest of the project; that does seem reasonable.
